src/notebooks/controllers/controllerLoader.ts

```diff
@@ -70,12 +69,7 @@ export class ControllerLoader implements IControllerLoader, IExtensionSyncActiva
     public loadControllers(refresh?: boolean | undefined): Promise<void> {
         if (!this.controllersPromise || refresh) {
             const stopWatch = new StopWatch();
-            // Cancel previous load
-            if (this.loadCancellationToken) {
-                this.loadCancellationToken.cancel();
-            }
-            this.loadCancellationToken = new vscode.CancellationTokenSource();
-            this.controllersPromise = this.loadControllersImpl(this.loadCancellationToken.token)
+            this.controllersPromise = this.loadControllersImpl()
```

Comment:
Cancellation token is no longer useful as the API is not pull based anymore, hence nothing can be cancelled.
I.e. nothing is async